src/main/haskell/kore/src/Kore/Step/Simplification/AndTerms.hs

```diff
@@ -347,8 +345,8 @@ addToolsArg
 addToolsArg = pure
 
 toExpanded
-    :: MetaOrObject level
-    =>  (  MetadataTools level StepperAttributes
+    :: (MetaOrObject level, SortedVariable variable, Show (variable level), Eq (variable level))
```

Comment:
Why the one-liner? It goes over the 90 character limit, which seems to be the latest agreed hard limit (though, from the Slack discussions, it is not clear to me yet if we still want to fit things in 80 chars and only go to 90 in exceptional cases, or 90 is the only limit that matters).

Reply:
Sorry, it seems that the style guide does not contain anything about 90 chars, it's still 80 chars, expandable to 100 in rare cases where it's needed.